main property target is wrong

**Describe the bug**

I can't use the package, because is marked as not found by my test runner.

**To Reproduce**

```sh
npm add jsdom-to-image
```

try to import the module as usual

```js
import { setGlobalOptions } from 'jsdom-to-image';

setGlobalOptions({
	connectOptions: {
		browserWSEndpoint: 'ws://localhost:9090',
	},
});

```

You get 

```sh
    Cannot find module 'jsdom-to-image' from 'jest.setup.ts'
```

**Expected behavior**

should import the module

**Additional context**

Problem comes from `package.json` **main** and **types** properties

```json
{
  "name": "jsdom-to-image",
  "version": "1.1.0",
  "description": "Generale image from JSDOM",
  "main": "dist/index.js",
  "types": "dist/index.d.ts",
}
```

They should be instead

```json
{
  "name": "jsdom-to-image",
  "version": "1.1.0",
  "description": "Generale image from JSDOM",
  "main": "dist/src/index.js",
  "types": "dist/src/index.d.ts",
}
```

--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
   ],
   "scripts": {
     "install:pre-commit-hooks": "pre-commit install --install-hooks --allow-missing-config -t pre-commit -t prepare-commit-msg -t commit-msg -f",
-    "build": "tsc",
+    "build": "tsc --project tsconfig.build.json",
     "lint": "eslint . --ext .ts --max-warnings=0",
     "prepare": "npm run build",
     "test": "jest",

--- a/tsconfig.build.json
+++ b/tsconfig.build.json
@@ -1,0 +1,6 @@
+{
+  "extends": "./tsconfig.json",
+  "include": [
+    "src"
+  ]
+}

